docker/pi0neerpat-packages/web/Dockerfile

```diff
@@ -0,0 +1,36 @@
+###########################################################################################
+# Builder: node
+###########################################################################################
+
+FROM node:14 as builder
+
+# Node
+ARG ENVIRONMENT
+ARG VERSION
+ARG REDWOOD_API_URL
+ARG APP_DOMAIN
+
+ENV ENVIRONMENT=$ENVIRONMENT
+ENV VERSION=$VERSION
+ENV REDWOOD_API_URL=$REDWOOD_API_URL
+ENV APP_DOMAIN=$APP_DOMAIN
+
+###########################################################################################
+# Runner: Nginx
+###########################################################################################
+
+FROM nginx as runner
+
+# Copy dist
+COPY --from=builder /app/web/dist /usr/share/nginx/html
+
+# Copy nginx configuration
+COPY web/config/nginx/default.conf /etc/nginx/conf.d/default.conf
+
+# List files
+RUN ls -lA /usr/share/nginx/html
```

Comment:
This was some legacy debugging code that I had in my initial example. Feel free to remove it if you'd like :)